spec/features/basket/cart_spec.rb

```diff
@@ -0,0 +1,164 @@
+describe 'Cart page' do
+  let(:book) { FactoryBot.create(:book) }
+
+  before do
+    visit book_path(book)
+    click_button I18n.t('button.add_to_cart')
+    sleep(1)
+    visit cart_path
+  end
+
+  context 'when increase the number of books' do
+    before do
+      within '#edit_order_item_1', match: :first do
+        find('button#increase_quantity').click
+      end
+    end
+
+    it { within('#edit_order_item_1', match: :first) { expect(find_field('order_item[quantity]').value).to eq '2' } }
+    it { expect(page).to have_css('#subtotal_1', text: OrderItem.first.subtotal) }
```

Comment:
```suggestion
    it { expect(page).to have_css('#subtotal_1', text: book.price * BOOK_AMOUNT) }
```
move book.price * BOOK_AMOUNT to constant